src/containers/MyNdla/MyProfile/components/MyPreferences.tsx

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { useState } from 'react';
+import styled from '@emotion/styled';
+import { colors, fonts, misc, spacing } from '@ndla/core';
+import { Heading, Text } from '@ndla/typography';
+import { RadioButtonGroup } from '@ndla/ui';
+import { useTranslation } from 'react-i18next';
+import { FeideUserApiType } from '../../../../interfaces';
+import { isStudent } from '../../Folders/util';
+
+type MyPreferencesProps = {
+  user: FeideUserApiType | undefined;
+};
+
+const PreferenceContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${spacing.normal};
+  max-width: 700px;
+`;
+
+const DisclaimerContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${spacing.small};
+`;
+
+const OptionContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: ${spacing.small};
+`;
+
+const StyledRadioButtonGroup = styled(RadioButtonGroup)`
+  gap: 0px;
+  max-width: 380px;
+  padding: 0;
+  > div {
+    box-sizing: content-box;
+    border-radius: ${misc.borderRadius};
+    border: 1px solid ${colors.brand.greyLight};
+    border-radius: 0px;
+    padding: ${spacing.small} ${spacing.normal};
+    border-color: ${colors.brand.light};
+    &:focus-within,
+    &[data-state='checked'] {
+      outline: 0px;
+      border-color: ${colors.brand.primary};
+      border-radius: 0px;
+      z-index: 1;
+    }
+    &:first-of-type {
+      border-top-left-radius: ${misc.borderRadius};
+      border-top-right-radius: ${misc.borderRadius};
+    }
+    &:not(:first-of-type) {
+      margin-top: -1px;
+    }
+    &:last-of-type {
+      border-bottom-left-radius: ${misc.borderRadius};
+      border-bottom-right-radius: ${misc.borderRadius};
+    }
+    > label {
+      ${fonts.sizes('16px', '24px')}
+      font-weight: ${fonts.weight.semibold};
+    }
```

Comment:
Testet litt og ser ut som du kan fjerne `border-radius` fra linje 47, 49 og 56. I tillegg til `border-color` på linje 51 for å forenkle littt her

Reply:
enig med linje 47 og 49. Om jeg fjerner de to andre så blir det faktisk forskjell i farge/border på fokus